### PR TITLE
snapshot/create: Fix subvol type setting to first value

### LINKED ISSUE #1278: consider  parent volume brick type which creating a snapshot bricks
### Observed behavior
subvolume types are considered as brick type in the current code

### Expected/desired behavior
subvolume brick type should be of parent volume brick  type

### PR DESCRIPTION
**Pull request by rafikc30**
we are setting the value of subvol type from an uninitialized
variable.

Fixes #1278

Signed-off-by: Mohammed Rafi KC <rkavunga@redhat.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
@rafikc30 please add  a Fixes tag to this PR, which can close the issue once PR is merged

**Comment by rafikc30** [comment]
@Madhu-1 How do I add fixes tag. Just update the commit message?

**Comment by Madhu-1** [comment]
Fixes: #1278

File: glusterd2/brick/types.go [hunk 1]
```diff
@@ -165,3 +165,13 @@ func (b *Brickinfo) Validate(check InitChecks, allLocalBricks []Brickinfo) error
 	// mandatory check that cannot be skipped forcefully
 	return isBrickInActiveUse(b.Path, allLocalBricks)
 }
+
+//BrickTypeToString converts BrickType to corresponding string
+func (b *Brickinfo) BrickTypeToString() string {
```
Comment (Madhu-1) [inline glusterd2/brick/types.go]: as we already have a function to convert brick type to string. is it possible to make use of [brick-type](https://github.com/gluster/glusterd2/blob/master/pkg/api/bricktype.go#L14)

Comment (Madhu-1) [reply glusterd2/brick/types.go]: discard this comment

**Pull request merged**